docs/ui/installation.rst

```diff
@@ -1,3 +1,8 @@
+Requirements
+============
+
+* Node.js_
+
```

Comment:
We had Node.js 'Requirements' twice in this file, so I moved it to the top as its own section, and now it appears only here.